auth-tokens/src/main/java/com/palantir/tokens/auth/BearerTokens.java

```diff
@@ -78,7 +78,7 @@ public static BearerToken fromPath(Path path) throws IOException {
         }
 
         LOGGER.warn("No token file found in any of the configured paths: {}.", paths);
-        return Optional.absent();
+        return Optional.empty();
```

Comment:
Most projects have been switching to Java `Optional` without a major version bump, on the basis it's a Java client break but not a wire break.